prereqs.py

```diff
@@ -13,13 +13,13 @@
 import functools
 
 python_ver = (3, 11)  # Python support for Windows on ARM64 requires v3.11 or later
-rust_ver = (1, 72)  # Ensure Rust version 1.69 or later is installed
+rust_ver = (1, 73)  # Ensure Rust version 1.69 or later is installed
```

Comment:
Probably should update the comment on the line also.

Reply:
Done